sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ParDoTest.java

```diff
@@ -1306,11 +1307,15 @@ public void testParDoShiftTimestamp() {
   @Category(NeedsRunner.class)
```

Comment:
Hm, shouldn't all these tests be RunnableOnService?.. I'm still confused about how exactly it differs from NeedsRunner.